TodoApi/appsettings.json

```diff
@@ -1,9 +1,14 @@
 {
-  "Logging": {
-    "LogLevel": {
+  "Serilog": {
+    "LogTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff} {Level} - {Message:lj}{NewLine}{Exception}",
+    "ElasticSearchUrl": "http://localhost:9200",
```

Comment:
Why is this set? What happens if I don't have it running?

Reply:
LogTemplate has default value and if we don't set elastic it will bypass

Reply:
Not the template the elastic search url shouldn't be set by default since it won't be running unless you run inside of the container.